shared/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/jjwt/JjwtEngineTest.groovy

```diff
@@ -14,18 +14,14 @@ class JjwtEngineTest extends Specification {
 
     def "readPrivateKey works"() {
         given:
-        def expected = """SunRsaSign RSA private CRT key, 2048 bits
```

Comment:
For some reason this test started to fail in GitHub for me.  I decided to refactor the test to not depend on `toString()`.